src/room.ts

```diff
@@ -86,8 +86,8 @@ export class Room {
 		}
 	}
 
-	public broadcast(severity: Severity, message: string) {
-		const msg: SystemMessage = {type: 'SYSTEM', message, severity};
+	public broadcast(severity: Severity, message: string, values: { [key: string]: string } | undefined) {
+		const msg: SystemMessage = (values) ? {type: 'SYSTEM', message, severity, values} : {type: 'SYSTEM', message, severity};
```

Comment:
No need for the check. If `values` is undefined then it's undefined.
```suggestion
		const msg: SystemMessage = {type: 'SYSTEM', message, severity, values};
```